### PR TITLE
Profile Règim especial

### PR DESCRIPTION
**Pull request by francescpuig7**
Profile Régim especial tariffs. With coefficients according climatic zones

### CODE REVIEW

File: enerdata/profiles/profile.py [hunk 1]
```diff
@@ -247,6 +261,28 @@ def get(cls, year, month):
                 conn.close()
             cls.down_lock.release()
 
+    @classmethod
+    def get_RE_cofs(self, start, end, tariff):
+        import pandas as pd
+        sheet_name = 'zona_{}'.format(tariff.climatic_zone)
+        df = pd.read_excel(
+            '/home/puig/codi/enerdata/enerdata/profiles/data/perfilesRE_por_zona_climatica.xlsx',
```
Comment (ecarreras) [inline enerdata/profiles/profile.py]: :scream:

File: enerdata/profiles/profile.py [hunk 2]
```diff
@@ -381,6 +417,10 @@ def estimate(self, tariff, balance):
             cofs = self.profile_class.get_range(
                 start, end - relativedelta(days=1)
             )
+
+        if tariff.code == 'RE':
+            cofs = self.profile_class.get_RE_cofs(start, end, tariff)
+
```
Comment (ecarreras) [inline enerdata/profiles/profile.py]: Don't do this, this is thought to assign `profile_class` and you should have a class with the necessary methods.

File: enerdata/profiles/profile.py (hunk 1 above)
Comment (francescpuig7) [reply enerdata/profiles/profile.py]: ups :rofl:

Comment (francescpuig7) [reply enerdata/profiles/profile.py]: Done via: https://github.com/gisce/enerdata/pull/115/commits/7610cafe2e5ea73f535a2c903d83cd808b2add71

File: enerdata/profiles/profile.py (hunk 2 above)
Comment (francescpuig7) [reply enerdata/profiles/profile.py]: done via: https://github.com/gisce/enerdata/pull/115/commits/de579f721e5db56c10a82841c560d3fa1a935eb2, https://github.com/gisce/enerdata/pull/115/commits/e98b1c8755d743efbf0e122989f491d6216f1b34, https://github.com/gisce/enerdata/pull/115/commits/0d9c1824ac0da38115a8cdb0dddfe593b9b61147, https://github.com/gisce/enerdata/pull/115/commits/31e98b5fd941b6e87e1e477e11fae7937e801508, https://github.com/gisce/enerdata/pull/115/commits/5a6a4bc1aa938a9a9d586c32b09b7f1660b28c07

**Pull request merged**